### DISCUSSION TITLE
MeshLab Server is missing from the MeshLab-2020.12 release for Windows

### DISCUSSION THREAD

**Discussion by alexturcea** [cnr-isti-vclab/meshlab#943] [General] [answered]
Is this expected? Was MeshLab Server deprecated?

**Comment by alemuntoni** [comment, marked as answer]
https://www.meshlab.net/#news
https://github.com/cnr-isti-vclab/meshlab/discussions/843
https://github.com/cnr-isti-vclab/meshlab/issues/911
